src/sentry/models/user.py

```diff
@@ -138,6 +138,11 @@ def get_full_name(self):
     def get_short_name(self):
         return self.username
 
+    def get_salutation_name(self):
+        name = self.name or self.username.split('@', 1)[0]
+        first_name = name.split(' ', 1)[0]
+        return first_name.capitalize()
```

Comment:
If i didn't have my name set and I signed up for sentry with my personal email, this would be `Jessmacqueen` which to me seems a little weird and maybe not better than just using my email address? Not totally sure of the context though so maybe it'd be fine

Reply:
Good point. For cases like yours I don't think there's much I could do to disentangle but I don't mind this scenario where you would get an email saying 'Hi Jessmcqueen.' One improvement I can add is for if you had jess.mcqueen@foo.com by doing an additional split to pull out just jess for that scenario.

Reply:
👍